app/models/patient.rb

```diff
@@ -459,6 +459,7 @@ def latest_transfer
     where(purged: false, monitoring: true, isolation: false, continuous_exposure: false)
       .where_assoc_not_exists(:assessments, symptomatic: true)
       .where_assoc_exists(:assessments) do
+        # Shouldn't this `where` be taking into account the patient's local time for date comparison?
```

Comment:
Question here and a few lines below: Should we be taking into account the patient's local time zone when making this date comparison

Reply:
I discussed this with @kylietmo when this feature was merged. We came to the conclusion that it need not.

* All dates in the scope except `user_curr_datetime` are in UTC already
* The scope is from the current user's perspective so based on their point of view, are they eligible?

Reply:
@Bialogs @kylietmo Is the requirement of `have reported within 7-9 days after their last date of exposure` meaning a range of 7-9 days after exposure in UTC or 7-9 days after exposure in the patient's local time zone?

Reply:
> All dates in the scope except user_curr_datetime are in UTC already

This part is not entirely true. The `last_date_of_exposure` is a user entered date that is not timestamped, so it's not saved in UTC time but rather whatever the user entered value was. 

> @Bialogs @kylietmo Is the requirement of have reported within 7-9 days after their last date of exposure meaning a range of 7-9 days after exposure in UTC or 7-9 days after exposure in the patient's local time zone?

After their entered `last_date_of_exposure` which is a user-entered date that is not converted.